src/lib/pubkey/pk_keys.h

```diff
@@ -145,6 +145,17 @@ class BOTAN_PUBLIC_API(2, 0) Public_Key : public virtual Asymmetric_Key {
       */
       virtual AlgorithmIdentifier algorithm_identifier() const = 0;
 
+      /**
+      * @return binary public key bits, with no additional encoding
+      *
+      * For key agreements this is an alias for PK_Key_Agreement_Key::public_value.
+      *
+      * Note: some algorithms (for example RSA) do not have an obvious encoding
+      * for this value due to having many different values, and thus not implement
+      * this function.
```

Comment:
Also mention that this function (should) always throw `Not_Implemented` in this case.